### DISCUSSION TITLE
Issue with job number

### DISCUSSION THREAD

**Discussion by WadeRolfe** [stayonhire/feedback#38] [Feedback / Issues]
Hi team, are we having any issues with changing the job number in new jobs? I am trying to change the current job number but it keeps reverting back to the listed number. It wont let me add the official job number from our quote.

![image](https://github.com/stayonhire/feedback/assets/141391069/de033351-a654-4112-8936-5263b7d2df6a)

**Comment by dylanhru** [comment]
Can confirm this is a bug and will get this fixed first thing this morning 👍

**Comment by twrayden** [comment]
Hi Wade, this should now be fixed, you might need to do a page refresh if your fix hasn't came through yet. Thanks for reporting
